cmd/installer/cli/install2.go

```diff
@@ -83,8 +82,8 @@ func Install2Cmd(ctx context.Context, name string) *cobra.Command {
 	var flags Install2CmdFlags
 
 	cmd := &cobra.Command{
-		Use:     "install",
-		Short:   fmt.Sprintf("Experimental installer for %s", name),
+		Use:   "install",
+		Short: fmt.Sprintf("Experimental installer for %s", name),
```

Comment:
we should change this description

Reply:
Could we do that after we merge this so as not to cause conflicts?